### DISCUSSION TITLE
Unknown Hard Error

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#2408] [提问] [answered]
如图
![XB37D X 0C{P8GLEHLVF6{P](https://github.com/Hex-Dragon/PCL2/assets/109946426/f2b125b1-f246-4dc6-86f2-932647431991)

PS. 群友的问题，要日志么

**Comment by Chiloven945** [comment]
这个不要日志咋看（

**Comment by XiaoLLiu617** [comment, marked as answer]
偶然性问题，并不造成实际影响，不必在意
